packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -225,8 +233,17 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
           {...field}
           id="completionDate"
           data-testid="completionDate"
-          minDate={new Date(watch('enrollmentDate'))}
+          minDate={(() => {
+            const date = new Date(watch('enrollmentDate'));
+            date.setDate(date.getDate() + 1);
+            return date;
+          })()}
           maxDate={new Date()}
+          isDisabled={(() => {
+            const d = new Date(watch('enrollmentDate'));
+            const today = new Date();
+            return d.setHours(0, 0, 0, 0) === today.setHours(0, 0, 0, 0);
```

Comment:
For this, it's probably actually better to use `dayjs` (or, alternatively, I guess `setUTCTime()`, but I always prefer libraries for date manipulation; this implementation can run into some weird edge-cases around DST shifts.

Reply:
Thanks @ibacher . Switched to using dayjs